conda-forge.yml

```diff
@@ -1,2 +1,4 @@
-provider: {linux_aarch64: default, linux_ppc64le: default}
+build_platform: {osx_arm64: osx_64}
```

Comment:
We still can't build this on arm64 because pyinstaller doesn't support python 3.8

Reply:
Ahhhh ok. I will remove that bit.

Reply:
xref: https://github.com/pyinstaller/pyinstaller/issues/4535#issuecomment-703067625